build/helm/edgemesh/charts/agent/values.yaml

```diff
@@ -28,6 +28,8 @@ modules:
     enable: true
     listenPort: 40001
     subNet: ""
+    socks5Proxy:
+      enable: true
```

Comment:
Consider adding the `listenPort` field so that users can use the `--set modules.edgeProxy.socksProxy.listenPort=10800` command to configure it.